src/Adapters/ADTAdapter.ts

```diff
@@ -199,7 +200,7 @@ export default class ADTAdapter implements IADTAdapter {
             );
         if (useCache) {
             return this.adtTwinCache.getEntity(
-                twinId,
+                buildCacheKey('adt_twin', twinId),
```

Comment:
why are we changing the cache keys for twins and other entities?

Reply:
to make the keys more explicit. do you have an idea in your mind for buildCacheKey method? I dont see much contribution of adding that buildCacheKey method otherwise

Reply:
I see. I was expecting the "buildCacheKey" to build this key.
 `adtInstanceIdentifier['hostName'] || adtInstanceIdentifier['id']`

That way we can use it when we add an item to the cache as well as reading it. BUT now I notice that we never actually call Set explicitly. We call get and pass a way to fetch the value if it's a cache miss. In that case I think your original approach is better.

Reply:
We don't really need the discriminator like `adt_twin` since every cache is separate so I think your original plan makes the most sense. Sorry for the confusion.

Reply:
no worries, reverting changes about key then